src/azure-cli-core/azure/cli/core/application.py

```diff
@@ -8,13 +8,13 @@
 import os
 import uuid
 import argparse
-from .parser import AzCliCommandParser
-from azure.cli._output import CommandResultItem
-import azure.cli.extensions
-import azure.cli._help as _help
-import azure.cli._logging as _logging
-from azure.cli._util import todict, CLIError
-from azure.cli._config import az_config
+from azure.cli.core.parser import AzCliCommandParser
+from azure.cli.core._output import CommandResultItem
+import azure.cli.core.extensions
```

Comment:
Yes, I have a note to move extensions out of core.
I'll do this as part of the work that defines exactly what goes into core.